src/main/kotlin/me/peckb/aoc/generators/CombinationsGenerator.kt

```diff
@@ -0,0 +1,36 @@
+package me.peckb.aoc.generators
+
+internal object CombinationsGenerator {
```

Comment:
not actually needed, but I thought it was, so I added it in for future use anyway :D